open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG

```diff
@@ -125,13 +125,20 @@ DEFINE
  Q                      = CHARACTER
  MAXERRORCODE           = 11
  TESTERRORCODE          = NUMBER
+ CONFIGPROGRAMVER       = CHARACTER
+
 END [DEFINE]
 
 SETUP
 [* Set default values
 *]
  DEBUGMODE=DEBUGMODEOFF
  Q=CTRLCHR(34)
+[* Configuration version must be in the following format:
+**   'a.b.c mm/dd/yyyy'
+**      a=major ver, b=minor ver, c=patch
+*]
+ CONFIGPROGRAMVER="1.1.0 10/20/2022"
```

Comment:
Maybe move the config file information to the top of the setup devision.  It should probably be the first thing defined in the setup.

Reply:
Moved.  Added additional comments and moved a couple other items (debug mode setting and quote ctrl value to the init procedure.